### ISSUE TITLE
`startOf('day')` behaves strangely for dates in `1969` and before.

### ISSUE THREAD

**Issue by mt-u** [kawanet/cdate#8]
Hi.

I report the mysterious behavior of `startOf('day')`.
When the code below is executed, the result of `1970` is the start date and time of the argument as expected, but the result of `1969` is the next day.
```typescript
import { cdate } from 'cdate';

const cdateJ = cdate().utcOffset(9).cdateFn();

function test(dateStr) {
  const d = new Date(dateStr);
  console.log(cdateJ(d).toDate(), cdateJ(d).format('YYYY-MM-DD HH:mm:ss'));

  const startOfDay = cdateJ(d).startOf('day');
  console.log(startOfDay.toDate(), startOfDay.format('YYYY-MM-DD HH:mm:ss'));
}

test('1970-01-01T23:59:59+09:00');

test('1969-01-01T23:59:59+09:00');
```

`1970` result:
```
Thu Jan 01 1970 23:59:59 GMT+0900 (日本標準時) '1970-01-01 23:59:59'
Thu Jan 01 1970 00:00:00 GMT+0900 (日本標準時) '1970-01-01 00:00:00'
```

`1969` result:
```
Wed Jan 01 1969 23:59:59 GMT+0900 (日本標準時) '1969-01-01 23:59:59'
Thu Jan 02 1969 00:00:00 GMT+0900 (日本標準時) '1969-01-02 00:00:00'
```

It seems to happen in `1969` and dates before that.
It does not occur when the unit passed to `startOf()` is `year` or `month`.
I've tried Firefox, Chrome and Edge, all with the same result.

Here is my environment:

- Windows 11 22H2 64bit
- Firefox: 113.0.2
- Chrome: 113.0.5672.127
- Edge: 113.0.1774.50
- Node.js: v18.14.2 (ubuntu:22.04 on Docker)

**Comment by kawanet** [comment]
Thank your for reporting this. I've just added tests to reproduce the issue which is not fixed yet though.

**Comment by kawanet** [comment]
cdate@0.0.6 published with the bugfix 🎉

https://www.npmjs.com/package/cdate/v/0.0.6

The bug was fixed with a single line patch 0c7134a. Thank you!
